src/main/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingController.kt

```diff
@@ -46,13 +59,51 @@ class EksternBehandlingController(private val pdlClient: PdlClient,
      */
     @PostMapping("finnes/flere-identer")
     @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
-    fun finnesBehandlingForPersonIdenter(@RequestParam("type") stønadstype: Stønadstype?,
-                                         @RequestBody personidenter: Set<String>): Ressurs<Boolean> {
+    fun finnesBehandlingForPersonIdenter(
+        @RequestParam("type") stønadstype: Stønadstype?,
+        @RequestBody personidenter: Set<String>
+    ): Ressurs<Boolean> {
         return finnesBehandlingFor(personidenter, stønadstype)
     }
 
-    private fun finnesBehandlingFor(personidenter: Set<String>,
-                                    stønadstype: Stønadstype?): Ressurs<Boolean> {
+    @GetMapping("hent/{personidenter}")
+    @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
+    fun erBehandlingerUtdaterteForPersonidenter(
+        @PathVariable personidenter: Set<String>
+    ): Ressurs<Boolean> {
+        if (personidenter.isEmpty()) {
+            return Ressurs.failure("Finner ikke identer til personen")
```

Comment:
Denne meldingen blir litt feil. "Minst en ident pakrevd for søk." eller noe lignende. Nå høres det ut som om det er søkeresultatet det er noe galt med.

Reply:
Godt poeng, endret denne